hadoop-ozone/dist/src/main/smoketest/recon/recon-api.robot

```diff
@@ -25,16 +25,26 @@ ${ENDPOINT_URL}       http://recon:9888
 ${API_ENDPOINT_URL}   http://recon:9888/api/v1
 
 *** Test Cases ***
-Recon REST API
+Recon OM APIs
+    Run Keyword if      '${SECURITY_ENABLED}' == 'true'     Kinit test user     testuser     testuser.keytab
+                        Execute                             ozone freon rk --numOfVolumes 1 --numOfBuckets 1 --numOfKeys 10 --keySize 1025
+                        Sleep               90s
```

Comment:
I'd rather avoid adding 2x1.5 minutes sleep to acceptance tests (one for secure, one for unsecure).

Reply:
We could use `Wait Until Keyword Succeeds` - https://github.com/apache/hadoop-ozone/blob/master/hadoop-ozone/dist/src/main/smoketest/basic/basic.robot#L35

Reply:
Thanks for the suggestions. I have fixed this.